doc/articles/features/ElevatedView.md

```diff
@@ -38,7 +38,7 @@ You can set the following properties:
 
 * `Elevation`: numeric number representing the level of the elevation effect. Typical values are between 5 and 30. The default is `0` - no elevation.
 * `ShadowColor`: By default the shadow will be `Black`, but you can set any other value. You can reduce the shadow effect by using the alpha channel [except Android]. On Android, the shadow color can only be changed since Android Pie (API 28+). The default is `Black` with alpha channel at 25%.
-* `Background`: The default is `Transparent`. Setting `null` will remove the shadow effect.
+* `Background`: The default is `null`, which does not show a shadow. You'll need to specify a color to avoid having a shadow below a transparent surface if the streching does not match the child.
```

Comment:
It is weird that the defaults for `Elevation` and `Background` result in not actually showing any shadow. 

We should set them to reasonable defaults that actually show an effect, otherwise the first impression (and possibly the last) is that the control just doesn't work.

Reply:
I agree, this PR is only about providing the current behavior of the control. We can adjust the defaults to make it work properly: https://github.com/unoplatform/uno/issues/7114